recipes/emio/all/test_package/CMakeLists.txt

```diff
@@ -0,0 +1,8 @@
+cmake_minimum_required(VERSION 3.8)
+project(test_package LANGUAGES CXX)
+
+find_package(emio REQUIRED)
+
+add_executable(test_package test_package.cpp)
+target_link_libraries(test_package PRIVATE emio::emio)
```

Comment:
This doesn't match the above target names!

Reply:
Hmm, I don't get it. The target name is everwhere the same: `test_package`

Reply:
No prob, it was related to the code that was removed, that apparently tried to define a different target name like only ``emio`` and not ``emio::emio``. but this is good.